src/explorer/weights/WeightSlider.js

```diff
@@ -33,11 +38,19 @@ export class WeightSlider extends React.Component<Props> {
   }
 }
 
+function convertWeightToSliderValue(weight: number): number {
+  return weight === 0 ? WEIGHT_SLIDER_MIN : Math.log2(weight);
```

Comment:
It’s confusing that `convertWeightToSliderValue` can return values that
are smaller than `WEIGHT_SLIDER_MIN`, and also (relatedly) that it’s
discontinuous. How about the following instead:

```js
  return Math.max(WEIGHT_SLIDER_MIN, Math.log2(weight));
```

When `weight` is `0`, its `log2` will be `-Infinity`, so everything
works out.

Reply:
Took this suggestion in my commit, although I think it's now unnecessary because I explicitly check the domain.